src/features/community/PrimaryIcon.tsx

```diff
@@ -0,0 +1,7 @@
+import styled from "@emotion/styled";
+import { IonIcon } from "@ionic/react";
+
+export const PrimaryIcon = styled(IonIcon)`
```

Comment:
Is this used?

Reply:
It was used for the earlier prototypes, to have the primary color applied to the icon like in this comment: https://github.com/aeharding/voyager/pull/713#issuecomment-1752310093

should I remove it?